pubsub-client/src/pubsub_client.rs

```diff
@@ -305,10 +305,9 @@ fn connect_with_retry(
     loop {
         let result = connect(url.clone()).map(|(socket, _)| socket);
         if let Err(tungstenite::Error::Http(response)) = &result {
-            if response.status() == reqwest::StatusCode::TOO_MANY_REQUESTS && connection_retries > 0
-            {
+            if response.status() == http::StatusCode::TOO_MANY_REQUESTS && connection_retries > 0 {
```

Comment:
Confirmed that `reqwest` re-exports `http::StatusCode`:
https://github.com/seanmonstar/reqwest/blob/44ac897f1ab35ba24a195927043d185d5cbb6912/src/lib.rs#L269